java/cdk/src/main/java/sleeper/cdk/stack/QueryStack.java

```diff
@@ -532,7 +534,7 @@ protected void setupWebSocketApi(LambdaCode queryJar, InstanceProperties instanc
     public Grant grantAccessToWebSocketQueryApi(IGrantable identity) {
         return Grant.addToPrincipal(GrantOnPrincipalOptions.builder()
                 .grantee(identity)
-                .actions(Collections.singletonList("execute-api:Invoke"))
+                .actions(List.of("execute-api:Invoke", "execute-api:ManageConnections"))
```

Comment:
It looks like this is intended to grant access for a client, but now it'll give them permissions to manage the connections. That seems like it shouldn't be necessary? Maybe this should be handled in setupWebSocketApi, like how grantManagementApiAccess is used to give access for the main queries Lambda?

Reply:
I have updated it to be in setupWebSockwtApi